src/Servers/Kestrel/Core/src/Internal/Http/HttpHeaders.Generated.cs

```diff
@@ -7925,6 +7925,15 @@ public unsafe bool TryHPackAppend(int index, ReadOnlySpan<byte> value, bool chec
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveOptimization | MethodImplOptions.AggressiveInlining)]
```

Comment:
I'm not sure `MethodImplOptions.AggressiveOptimization` does anything in practice.